pkg/operator/certrotationcontroller/externalloadbalancer.go

```diff
@@ -18,6 +18,7 @@ func (c *CertRotationController) syncExternalLoadBalancerHostnames() error {
 	hostname := infrastructureConfig.Status.APIServerURL
 	hostname = strings.Replace(hostname, "https://", "", 1)
 	hostname = hostname[0:strings.LastIndex(hostname, ":")]
+	hostname = strings.Replace(hostname, "api-int.", "api.", 1)
```

Comment:
seems like this needs to go in first, then https://github.com/openshift/installer/pull/1633, then the other change below

no?